test/test.js

```diff
@@ -156,52 +150,42 @@ describe('Users', () => {
       }
 
       AddUser(user, () => {
-        AddInvalidUser(invalidUser, () => {
-          done()
-        })
+        AddInvalidUser(invalidUser, [{title: 'Email already exists'}], done)
       })
     })
 
   it('GET /users/:username | Should give us the users info', (done) => {
     AddUser(user, () => {
       chai.request(server).get(`/users/${user.username}`).end((err, res) => {
-        should.not.exist(err)
-        res.should.have.status(202)
-        res.body.should.be.a('object')
-        res.body.should.have.property('errors')
-        res.body.errors.length.should.be.eql(0)
-        res.body.should.have.property('data')
-        res.body.data.should.have.property('name')
-        res.body.data.should.have.property('name').eql(`${user.name}`)
-        res.body.data.should.have.property('username')
-        res.body.data.should.have.property('username').eql(`${user.username}`)
-        res.body.data.should.have.property('datecreated')
-        res.body.data.should.have.property('description')
-        res.body.data.should.have.property('posts')
-        res.body.data.posts.should.be.a('array')
-        res.body.data.posts.length.should.be.eql(0)
-        res.body.data.should.have.property('followers')
-        res.body.data.followers.should.be.a('array')
-        res.body.data.followers.length.should.be.eql(0)
-        res.body.data.should.have.property('following')
-        res.body.data.following.should.be.a('array')
-        res.body.data.following.length.should.be.eql(0)
-        done()
+        VerifyValidResponse(err, res, () => {
+          res.body.data.should.have.property('name')
+          res.body.data.should.have.property('name').eql(`${user.name}`)
+          res.body.data.should.have.property('username')
+          res.body.data.should.have.property('username').eql(`${user.username}`)
+          res.body.data.should.have.property('datecreated')
+          res.body.data.should.have.property('description')
+          res.body.data.should.have.property('posts')
+          res.body.data.posts.should.be.a('array')
+          res.body.data.posts.length.should.be.eql(0)
+          res.body.data.should.have.property('followers')
+          res.body.data.followers.should.be.a('array')
+          res.body.data.followers.length.should.be.eql(0)
+          res.body.data.should.have.property('following')
```

Comment:
how about destructuring to get only data?

Reply:
Can't cause `verifyValidResponse` needs the entire object, unless you mean destructure it after it?

Reply:
I mean destructuring on line 160